File: client/iih_stub/iih.ci_stub/iih.ci.mr/cimrvt/i/ICiVitalSignsService.cs
using iih.ci.mr.cimrpatsigns.d;
namespace iih.ci.mr.cimrvt.i
{
    public interface ICiVitalSignsService
    {
        VitalSignsDTO[] GetVitalSignsDTOList(string Id_ent, string BeginDate, string EndDate); 
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/i/ICiVitalSignsService.java
package iih.ci.mr.i;


import iih.ci.mr.cimrpatsigns.d.VitalSignsDTO;
import xap.mw.core.data.BizException;

public interface ICiVitalSignsService {
	
	/*
	 * 
	 * @param Id_ent
	 * @return BeginDate
	 * @throws EndDate
	 */
	public abstract VitalSignsDTO[] GetVitalSignsDTOList(String Id_ent,String BeginDate,String EndDate)
			throws BizException;
}
